examples/webgl-vector-tiles.js

```diff
@@ -6,39 +6,62 @@ import View from '../src/ol/View.js';
 import WebGLVectorTileLayerRenderer from '../src/ol/renderer/webgl/VectorTileLayer.js';
 import {Fill, Icon, Stroke, Style, Text} from '../src/ol/style.js';
 import {asArray} from '../src/ol/color.js';
-import {packColor} from '../src/ol/webgl/styleparser.js';
+import {packColor, parseLiteralStyle} from '../src/ol/webgl/styleparser.js';
 
 const key =
   'pk.eyJ1IjoiYWhvY2V2YXIiLCJhIjoiY2t0cGdwMHVnMGdlbzMxbDhwazBic2xrNSJ9.WbcTL9uj8JPAsnT9mgb7oQ';
 
+const result = parseLiteralStyle({
+  'fill-color': ['get', 'fillColor'],
+  'stroke-color': ['get', 'strokeColor'],
+  'stroke-width': ['get', 'strokeWidth'],
+  symbol: {
+    symbolType: 'circle',
+    size: 8,
+    color: '#777',
+  },
```

Comment:
Not important to change anything here, but this is what this style literal would look like when passed to the layer:
```js
{
  'fill-color': ['get', 'fillColor'],
  'stroke-color': ['get', 'strokeColor'],
  'stroke-width': ['get', 'strokeWidth'],
  'circle-radius': 8,
  'circle-fill-color': '#777',
}
```

Reply:
Yes, hopefully the webgl renderers can align with this format soon! Thanks for the hint